### PR TITLE
Updated doc links

### PR DESCRIPTION
**Pull request by joel-hamill**
I have update the linkage to the new docs.mesosphere paths.

### CODE REVIEW

File: bin/install/legacy/install-legacy-optout-dcos-cli.sh [hunk 1]
```diff
@@ -98,7 +98,7 @@ if [[ $VIRTUAL_ENV_PATH =~ \  ]];
 fi
 DCOS_URL=${ARGS[1]}
 
-command -v virtualenv >/dev/null 2>&1 || { echo "Cannot find virtualenv. You may need to install it by following the documentation at https://docs.mesosphere.com/install/cli/#linux. Aborting."; exit 1; }
+command -v virtualenv >/dev/null 2>&1 || { echo "Cannot find virtualenv. You may need to install it by following the documentation at shttps://dcos.io/docs/usage/cli/install/. Aborting."; exit 1; }
```
Comment (tamarrow) [inline bin/install/legacy/install-legacy-optout-dcos-cli.sh]: extra s before https

**Comment by tamarrow** [comment]
Thanks! Will merge when the docs go live.

**Comment by tamarrow** [comment]
@joel-hamill can you revert your changes in the test files? They are using an old universe that doesnt have the new naming, which is why the tests are failing

**Comment by joel-hamill** [comment]
@tamarrow The only changes that I see to test were the doc link from `http` to `https`? Are there more? http://cl.ly/3e0B0N3X3Z1c

**Comment by jsancio** [comment]
@joel-hamill @tamarrow Do we need to merge this ASAP?

**Comment by joel-hamill** [comment]
@jsancio old links will still work for now because there are redirects. i'll let tamar make the call on when to merge.

**Comment by tamarrow** [comment]
I didn't merge this in the release last night, because the new links didnt work properly (ie they didnt redirect to /docs/1.7/usage/cli correctly). Merging to master now so it'll go in the next release.

**Pull request merged**